### PR TITLE
feat(ArchLinux): add presence

### PR DESCRIPTION
**Pull request by critamine**
**Describe the changes in this pull request:**

- First coding project and PR!
- Added basic presence support for ArchLinux.org

<details>
<summary> Proof showing the creation/modification is working as expected </summary>

archLinux

</details>

### CODE REVIEW

**Review by critamine — COMMENTED** [review]
Corrected version in metadata.json

File: websites/A/ArchLinux/presence.ts [hunk 1]
```diff
@@ -0,0 +1,183 @@
+const presence = new Presence({
+		clientId: "929881116679237653"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	hostName = location.hostname,
+	webPath = location.pathname,
+	path1 = webPath.split("/")[1];
+presence.on("UpdateData", async () => {
```
Comment (Bas950) [inline websites/A/ArchLinux/presence.ts:8]:
```suggestion
	path1 = webPath.split("/")[1];

presence.on("UpdateData", async () => {
```

File: websites/A/ArchLinux/presence.ts [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+	type dt = {
+		[hostName: string]: {
+			[webPath: string]: {
+				[data: string]: string;
+			};
+		};
+	};
+
+	const archData: dt = {
```
Comment (Bas950) [inline websites/A/ArchLinux/presence.ts:22]:
```suggestion
	const archData: {
		[hostName: string]: {
			[webPath: string]: {
				[data: string]: string;
			}
		}
	} = {
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by critamine — COMMENTED** [review]
Thank you for taking the time to review this and offer feedback.

**Review by critamine — COMMENTED** [review]
Changes reviewed

Not too sure what I'm doing here. (With GitHub)

**Review by critamine — COMMENTED** [review]
I think I unviewed/un-reviewed a file here. Fixing that. Apologies for cluttering up the PR! New to this.

File: websites/A/ArchLinux/presence.ts [hunk 3]
```diff
@@ -0,0 +1,181 @@
+const presence = new Presence({
+		clientId: "929881116679237653"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	hostName = location.hostname,
+	webPath = location.pathname,
```
Comment (EncryptedDev) [inline websites/A/ArchLinux/presence.ts:6]:
```suggestion
	{ pathname, hostname } = document.location,
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/A/ArchLinux/presence.ts (hunk 3 above)
Comment (critamine) [reply websites/A/ArchLinux/presence.ts]: Ohhh! Cool! Thank you!

**Review by critamine — COMMENTED** [review]
Changes reviewed. Thank you so much for looking!

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**